File: 7kyu/ShortestWordInString.java
/**
 * Shortest Word - Java
 *
 * Simple, given a string of words, return the length of the shortest word(s).
 * String will never be empty and you do not need to account for different data
 * types.
 * 
 * @author Unai de la O
 */

 import java.util.Arrays;

 public class ShortestWordInString {
   public static int findShort(String s) {
     String[] words = s.split(" ");
     int[] lengthsArr = new int[words.length];

     for (int i = 0; i < lengthsArr.length; i++) {
       lengthsArr[i] = words[i].length();
     }
     Arrays.sort(lengthsArr);
     return lengthsArr[0];
   }
 }

File: 7kyu/ShortestWordInString.cs
/*
 * Shortest Word - C#
 *
 * Simple, given a string of words, return the length of the shortest word(s).
 * String will never be empty and you do not need to account for different data
 * types.
 *
 * Author: Unai de la O
 */
 using System;
 using System.Linq;

 public class ShortestWordInString
 {
   public static int FindShort(string s)
   {
     string[] words = s.Split(' ');
     int[] lengthsArr = new int[words.Length];

     for (int i = 0; i < lengthsArr.Length; i++)
     {
       lengthsArr[i] = words[i].Length;
     }
     return lengthsArr.Min();
   }
 }
